config.yml

```
@@ -13,4 +13,15 @@ client:
     path: "vue/src/store"
 faucet:
   name: bob
-  coins: ["5token", "100000stake"]
+  coins: ["5mand", "100000stake"]
+genesis:
+  app_state:
+    staking:
+      params:
+        unbonding_time: "0.001s"
+    bank:
+      params:
+        send_enabled: [{"denom": "stake","enabled": false}]
+    mint:
+      params:
+        mint_denom: "mand"
```

Please add an empty line for sanity
```suggestion
        mint_denom: "mand"
        
```